src/BrightScriptDebugSession.spec.ts

```diff
@@ -366,15 +366,15 @@ describe('Debugger', () => {
 
         it('remaps to debug folder when specified', () => {
             (session as any).launchArgs = {
-                debugRootDir: path.normalize('/src'),
+                sourceDirs: [path.normalize('/src')],
```

Comment:
Can you add another test that verifies `debugRootDir` is correctly converted to `sourceDirs`?